pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseSingleStageBrokerRequestHandler.java

```diff
@@ -306,139 +306,71 @@ protected BrokerResponse handleRequest(long requestId, String query, SqlNodeAndO
     }
   }
 
+  static class CompileResult {
```

Comment:
(minor) This can be `private` since it is just a helper wrapper class

Reply:
Done